src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/SnackBar.uwp.cs

```diff
@@ -34,11 +34,16 @@ class SnackBar
 			return null;
 		}
 
-		internal void Show(Forms.Page page, SnackBarOptions arguments)
+		internal void Show(Forms.VisualElement visualElement, SnackBarOptions arguments)
 		{
 			var snackBarLayout = new SnackBarLayout(arguments);
-			var pageControl = Platform.GetRenderer(page).ContainerElement.Parent;
-			var grid = FindVisualChildByName<Border>(pageControl, "BottomCommandBarArea").Parent as Grid;
+			var pageControl = Platform.GetRenderer(visualElement).ContainerElement.Parent;
+			var grid = FindVisualChildByName<Border>(pageControl, "BottomCommandBarArea")?.Parent as Grid;
+			if (grid is null)
+			{
+				return;
```

Comment:
Should there be some exception here, since we are not displaying SnackBar?

Reply:
UWP, for now, doesn't support anchor view. so I think it is better to return null rather than throw an exception